JeMPI_Apps/JeMPI_LibShared/src/main/java/org/jembi/jempi/shared/utils/AuditTrailUtil.java

```diff
@@ -0,0 +1,59 @@
+package org.jembi.jempi.shared.utils;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.jembi.jempi.shared.kafka.MyKafkaProducer;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jembi.jempi.shared.models.AuditEvent;
+import org.jembi.jempi.shared.models.GlobalConstants;
+
+import static org.jembi.jempi.shared.utils.AppUtils.OBJECT_MAPPER;
+
+
+public final class AuditTrailUtil {
```

Comment:
Could you rename this to something like `AuditTrailBridge` as Util classes don't usually get instantiated